src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -189,5 +189,34 @@ void conjured_update_qualityDropsTwiceAsFastAsNormalItemsWhenSellInIsPassed0() {
         assertEquals(28, app.items[0].quality);
     }
 
+    @Test
+    void redWine_update_qualityDropsThreeTimesSlowerThanNormalItems() {
```

Comment:
This is a nice test. Update quality 3 times and assert the quality only decremented one step.  👍